src/extensions/cellpack/model.ts

```diff
@@ -707,6 +625,20 @@ async function loadPackings(plugin: PluginContext, runtime: RuntimeContext, stat
                         await loadMembrane(plugin, packings[i].name, state, params);
                     }
                 }
+            } else if (params.membrane === 'spheres'){
+                if (packings[i].mb) {
+                    let nSpheres =  packings[i].mb!.positions!.length / 3;
+                    console.log('ok mb ', nSpheres);
+                    for (let j = 0;j < nSpheres;j++) {
+                        await state.build()
```

Comment:
- How many spheres would be here? It would be much more efficient to have a ``CreateSpheres`` transform which would take a list of vectors (Just using ``PD.Value<Vec3[]>``).
- Also, when applying multiple transforms, it is much more efficient to do:

```ts
const update  = state.build().toRoot();
for (let j = 0;j < nSpheres;j++) {
  update.apply(...);
}
await update.commit();
```